src/langsmith/administration-overview.mdx

```diff
@@ -160,17 +168,19 @@ RBAC (Role-Based Access Control) is a feature that is only available to Enterpri
 
 Roles are used to define the set of permissions that a user has within a workspace. There are three built-in system roles that cannot be edited:
 
-* `Admin` - has full access to all resources within the workspace
-* `Viewer` - has read-only access to all resources within the workspace
-* `Editor` - has full permissions except for workspace management (adding/removing users, changing roles, configuring service keys)
+- [Workspace Admin](/langsmith/rbac#workspace-admin) has full access to all resources within the workspace.
+- [Workspace Editor](/langsmith/rbac#workspace-editor) has full permissions except for workspace management (adding/removing users, changing roles, configuring service keys).
+- [Workspace Viewer](/langsmith/rbac#workspace-viewer) has read-only access to all resources within the workspace.
```

Comment:
@bvs-langchain I added "Workspace" to the front of these roles to match what I'd done in the operations reference, but now I'm realizing that they are Admin, Viewer, and Editor in the UI. Should I remove "Workspace" from all the roles that include that in this PR, or do you think it's good to keep in order to differentiate from the Org roles?